scalafix-core/shared/src/main/scala/scalafix/internal/util/PrettyType.scala

```diff
@@ -347,29 +348,23 @@ class PrettyType private (table: SymbolTable, shorten: QualifyStrategy) {
       }
   }
 
+  def unwrapRepeatedType(tpe: s.Type): s.Type = tpe match {
+    case s.RepeatedType(targ) =>
+      // Workaround for https://github.com/scalameta/scalameta/issues/1497
+      s.TypeRef(
+        prefix = s.NoType,
+        symbol = "scala.collection.Seq#",
+        typeArguments = targ :: Nil
+      )
+    case targ =>
+      targ
+  }
```

Comment:
So this is where this ticket comes from!

Reply:
Yup. `RepeatedType` can appear in both `Init` position and also for `Defn.Def.decltpe`

```scala
case class A(a: AnyRef*)
```
currently produces a `val a: AnyRef*` method symbol